dataactvalidator/scripts/loadFile.py

```diff
@@ -129,6 +129,7 @@ def loadDomainValues(basePath, localProgramActivity = None):
 
 
 if __name__ == '__main__':
+    configure_logging()
```

Comment:
This might help explain: If you were in a `python` session and called `loadFile` directly, you'd get the default logging settings of your environment (usually, writing INFO/WARNING/ERROR to the console), regardless of what files we'd "normally" write to.